### ISSUE TITLE
This zip is not a Magisk Module

### ISSUE THREAD

**Issue by sharukhmohammed** [topjohnwu/magisk-module-template#42]
[Magic Font Lato 1.0.zip](https://github.com/topjohnwu/magisk-module-template/files/1466843/Magic.Font.Lato.1.0.zip)
I have this simple magisk module to replace the system font. 
I have tried countless times to rectify the error, started from scratch and still end up here.
I am on latest Magisk and Magisk Manager at the time of writing.
I'll be really happy if you look into the issue, I have attached the module as a ZIP.

**Comment by dark-1** [comment]
Try this : [Magic.Font.Lato.1.0.zip](https://github.com/topjohnwu/magisk-module-template/files/1466958/Magic.Font.Lato.1.0.zip)
And Report plz.... 👍

**Comment by ShadySquirrel** [comment]
Explanation: files need to be in the root of the zip, so compress files inside of module folder, not module folder.

**Comment by dark-1** [comment]
>Explanation: files need to be in the root of the zip, so compress files inside of module folder, not module folder.

That's what I did , but was not sure if that was the problem so gave that zip , along with it cleaned some unneeded files.

**Comment by ShadySquirrel** [comment]
> That's what I did , but was not sure if that was the problem so gave that zip , along with it cleaned some unneeded files.

Yeah, and I told OP what you did :D

**Comment by Zackptg5** [comment]
Also, the updater-script needs to contain only "#MAGISK" (case sensitive) or you'll get that message in magisk manager

**Comment by sharukhmohammed** [comment]
@dark-1 Thanks, man, ZIP works now. Can you specify which files have you cleaned?
@ShadySquirrel Thanks for your explanation, I appreciate! 👍 
@Zackptg5 Yes, the updater-script contains only '#MAGISK'

**Comment by dark-1** [comment]
@sharukhmohammed ,

Nice to know that the _ZIP_ worked 👍 .
Sorry that I might have mis-informed that I `Cleaned some Files` actually I did not touch any files , but just removed not required file.

This is what I did : 
1. Removed all `.DS_Store` files , I am guessing you are working on MAC and that file is in every nook n corner , each n every folder. [ Kindly see that they are not in flash'able ZIP ]
2. Removed `.git` folder , Not required while Flashing.  [ Kindly see that they are not in flash'able ZIP ]
3. As per your [config.sh](https://github.com/sharukhmohammed/magic-font-lato/blob/1400/config.sh) , You do not need the folder `common` and it's contends. So might as well remove them. [ Your Thought full Choice , This just a recommendation ]
4. In your `magic-font-lato` folder , select the required folders and files and zip them,
Required folders for you : `META-INF`, `system`.
Required files for you : `config.sh`, `module.prop`, `README.md`.
[I have never used a MAC pc so I have no idea how you will be doing it , hence sorry for the same]
5. EnJoY 😄 and play with it. 

Best Regards. 👍
